tests/integration/kubernetes/lib.sh

```diff
@@ -94,11 +94,41 @@ assert_pod_fail() {
 	! k8s_create_pod "$container_config" || /bin/false
 }
 
+
+# Check the pulled rootfs on host for given node and sandbox_id
+#
+# Parameters:
+#	$1 - the k8s worker node name
+#	$2 - the sandbox id for kata container
+#	$3 - the expected count of pulled rootfs
+#
+assert_rootfs_count() {
+	local node="$1"
+	local sandbox_id="$2"
+	local expect_count="$3"
+	local allrootfs=""
+	# Max loop 3 times to get all pulled rootfs for given sandbox_id
+	for _ in {1..3}
```

Comment:
This looks fragile (same comment for `get_node_kata_sandbox_id()`). It might be better to do a timed wait for up to a much longer amount of time (60 seconds, say?) to allow for slow network, remote server, _etc_.

Reply:
Fixed.